addons/payment_xendit/static/src/js/payment_form.js

```diff
@@ -138,7 +138,13 @@ paymentForm.include({
      */
     _xenditHandleResponse(err, token, processingValues) {
         if (err) {
-            this._displayErrorDialog(_t("Payment processing failed"), err.message);
+            let errMessage = err.message;
+
+            if (err.error_code === 'API_VALIDATION_ERROR') {  // Invalid user input
+                let errObj = err.errors[0]
+                errMessage = errObj.message // Wrong field format
```

Comment:
Now we do not need `errObj`
```suggestion
                errMessage = err.errors[0].message; 
```